### PR TITLE
Cumulative Commerce PR post-RCP

### PR DESCRIPTION
**Pull request by 3ch023**
This PR is a combination of several commerce PR done during RCP.
Code review was done on each PR merged in 'ccd' branch. QA was done as well. (and will be extensively done by us on this PR second time)

This PR is still open for a second round of code reviews. Please try to review within the next 3 days, since we would like to merge the changes before the next RCP (12 Dec).

The main focus for Milo community is to check for regressions (will be done on stage branch as well, but assuming the size of the PR it is better to do at the earlier stage).

Resolves: merge commerce RCP changes.

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?martech=off
- After: https://ccd--milo--adobecom.hlx.live/docs/library/kitchen-sink/merch-card?martech=off

### CODE REVIEW

File: libs/features/mas/test/mocks/pandora-fetch.js
Comment (npeltier) [inline libs/features/mas/test/mocks/pandora-fetch.js:1]: do we still need this?

File: libs/features/mas/test/mocks/react.js
Comment (npeltier) [inline libs/features/mas/test/mocks/react.js:1]: same, do we still need this?

**Review by npeltier — APPROVED** [review]
already checked most of it in PRs -> ccd, but looks good again :) (with few additional remarks i hadn't spotted before)

File: libs/features/mas/test/mocks/react.js
Comment (yesil) [reply libs/features/mas/test/mocks/react.js:1]: I'm not sure, but we should remove in a separate PR after checking.

**Review by yesil — APPROVED** [review]

File: libs/features/mas/src/lana.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    if (type === 'object') {
+        if (value instanceof Error) return value.message;
+
+        if (typeof value.originatingRequest === 'string') {
+            const { message, originatingRequest, status } = value;
+            return [message, status, originatingRequest]
+                .filter(Boolean)
+                .join(' ');
+        }
```
[the remark this answers is not in the document]
Comment (3ch023) [reply libs/features/mas/src/lana.js:38]: it's possible, but I will address smaller changes in a separate PR, i'd like to touch this one as less as possible and only fix serious issues.

File: libs/features/mas/test/mocks/pandora-fetch.js
Comment (3ch023) [reply libs/features/mas/test/mocks/pandora-fetch.js:1]: will check in a follow up

**Comment by afmicka** [comment]
@yesil @3ch023 @npeltier @Axelcureno @Roycethan 

We have few differences that might be because the branch is not up-to-date with stage:
1. CTA blue color is different (not aligned with https://github.com/adobecom/milo/pull/3221/files) 
2. One of the icons is affected in the. merch-card on DC
https://main--dc--adobecom.hlx.live/acrobat/how-to/share-pdf-online?milolibs=ccd
Screenshot 2024-12-09 at 16 21 23
Screenshot 2024-12-09 at 16 21 18


Otherwise, regressions look ok on my end. @Roycethan Could you please double-check from your side? If all good, lets try to add this PR to the current Stage to Main train

**Review by Axelcureno — APPROVED** [review]

**Comment by Roycethan** [comment]
@yesil @3ch023  Checked from my side...Plz check Nala and Code-cov failures to proceed further.. cc @afmicka

**Comment by afmicka** [comment]
@Roycethan for Nala, it needs the most recent stage update. The failing tests were just fixed in https://github.com/adobecom/milo/pull/3335

**Comment by 3ch023** [comment]
CTA color PR wasn't done correctly, it didn't change colour in src file
https://github.com/adobecom/milo/pull/3221/files

I will address it in the follow up

**Pull request merged**